### PR TITLE
Don't kill no pipelines :-)

### PR DESCRIPTION
**Pull request by ckadner**
**Description of your changes:**

The word "execute" conjures up images of people getting their life cut short involuntarily.

This PR suggests to use the terms "run" and "running" a pipeline instead. At least in the headings.

https://github.com/kubeflow/kfp-tekton/pull/211#discussion_r449325481

Also, title-casing headings.

/cc @Tomcli

### CODE REVIEW

File: samples/kfp-user-guide/README.md [hunk 1]
```diff
@@ -250,8 +268,10 @@ kfp run get bb96363f-ec0d-4e5a-9ce9-f69a485c2d94
 ```
 
 
-### 4. Optional: Execute Tekton pipelines without using Kubeflow Pipelines engine
-For all the compiled Kubeflow pipelines, we can extract them into `.yaml` format and run them directly on Kubernetes using the Tekton CRD. However, this method should **only use for Tekton development** because Kubeflow Pipelines engine cannot track these pipelines that run directly on Tekton.
+### 4. Optional: Run Tekton Pipelines Without Using the Kubeflow Pipelines Engine
+
+For all the compiled Kubeflow pipelines, we can extract them into `.yaml` format and run them directly on Kubernetes using the Tekton CRD.
```
Comment (animeshsingh) [inline samples/kfp-user-guide/README.md:273]:
```suggestion
For all the compiled Kubeflow pipelines, we can extract them into `.yaml` format and run them directly on Tekton using the Tekton CRD.
```

File: samples/kfp-user-guide/README.md [hunk 2]
```diff
@@ -21,24 +21,28 @@ def echo_pipeline(
...
-  - [Execute Pipelines](#execute-pipelines)
-    - [1. Execute pipelines using the Kubeflow Pipeline user interface.](#1-execute-pipelines-using-the-kubeflow-pipeline-user-interface)
-    - [2. Execute pipelines using the `kfp_tekton.TektonClient` in Python](#2-execute-pipelines-using-the-kfp_tektontektonclient-in-python)
-    - [3. Execute pipelines using the `kfp` bash command line tool](#3-execute-pipelines-using-the-kfp-bash-command-line-tool)
-    - [4. Optional: Execute Tekton pipelines without using Kubeflow Pipelines engine](#4-optional-execute-tekton-pipelines-without-using-kubeflow-pipelines-engine)
-
-## Compile Pipelines
-
-### 1. Compile pipelines using the `kfp_tekton.compiler.TektonCompiler` in Python
+  - [Compiling Pipelines](#compiling-pipelines)
```
Comment (animeshsingh) [inline samples/kfp-user-guide/README.md:28]:
```suggestion
  - [Compile Pipelines](#compile-pipelines)
```

File: samples/kfp-user-guide/README.md [hunk 3]
```diff
@@ -21,24 +21,28 @@ def echo_pipeline(
...
+    - [1. Compile Pipelines Using the `kfp_tekton.compiler.TektonCompiler` in Python](#1-compile-pipelines-using-the-kfp_tektoncompilertektoncompiler-in-python)
+    - [2. Compile Pipelines Using the `dsl-compile-tekton` Bash Command Line Tool](#2-compile-pipelines-using-the-dsl-compile-tekton-bash-command-line-tool)
+  - [Uploading Pipelines](#uploading-pipelines)
```
Comment (animeshsingh) [inline samples/kfp-user-guide/README.md:31]:
```suggestion
  - [Upload Pipelines](#upload-pipelines)
```

File: samples/kfp-user-guide/README.md [hunk 4]
```diff
@@ -21,24 +21,28 @@ def echo_pipeline(
...
+    - [1. Upload Pipelines with the Kubeflow Pipeline User Interface](#1-upload-pipelines-with-the-kubeflow-pipeline-user-interface)
+    - [2. Upload Pipelines Using the `kfp_tekton.TektonClient` in Python](#2-upload-pipelines-using-the-kfp_tektontektonclient-in-python)
+    - [3. Upload Pipelines Using the `kfp` Bash Command Line Tool](#3-upload-pipelines-using-the-kfp-bash-command-line-tool)
+  - [Running Pipelines](#running-pipelines)
```
Comment (animeshsingh) [inline samples/kfp-user-guide/README.md:35]:
```suggestion
  - [Run Pipelines](#run-pipelines)
```

File: samples/kfp-user-guide/README.md [hunk 5]
```diff
@@ -21,24 +21,28 @@ def echo_pipeline(
...
+    - [1. Run Pipelines with the Kubeflow Pipeline User Interface](#1-run-pipelines-with-the-kubeflow-pipeline-user-interface)
+    - [2. Run Pipelines Using the `kfp_tekton.TektonClient` in Python](#2-run-pipelines-using-the-kfp_tektontektonclient-in-python)
+    - [3. Run Pipelines Using the `kfp` Bash Command Line Tool](#3-run-pipelines-using-the-kfp-bash-command-line-tool)
+    - [4. Optional: Run Tekton Pipelines Without Using the Kubeflow Pipelines Engine](#4-optional-run-tekton-pipelines-without-using-the-kubeflow-pipelines-engine)
+
+
+## Compiling Pipelines
```
Comment (animeshsingh) [inline samples/kfp-user-guide/README.md:42]:
```suggestion
## Compile Pipelines
```

File: samples/kfp-user-guide/README.md [hunk 6]
```diff
@@ -50,15 +54,18 @@ In addition, we can put the above python code into a python `__main__` function.
...
+
 The kfp-tekton SDK also comes with a bash command line tool for compiling Kubeflow pipelines. Here, we need to store the above `echo_pipeline` example into a python file called `echo_pipeline.py` and run the below bash command. The `--output` supports the following formats: `[.tar.gz, .tgz, .zip, .yaml, .yml]`
 ```shell
 dsl-compile-tekton --py echo_pipeline.py  --output pipeline.yaml
 ```
 
-## Upload Pipelines
 
-### 1. Upload pipelines using the Kubeflow Pipeline user interface.
+## Uploading Pipelines
```
Comment (animeshsingh) [inline samples/kfp-user-guide/README.md:65]:
```suggestion
## Upload Pipelines
```

File: samples/kfp-user-guide/README.md [hunk 7]
```diff
@@ -137,9 +148,11 @@ kfp pipeline upload-version -p <existing_pipeline_id> -v new_pipeline_version ec
 # Uploaded at   2020-07-02T17:58:05+00:00
 ```
 
-## Execute Pipelines
 
-### 1. Execute pipelines using the Kubeflow Pipeline user interface.
+## Running Pipelines
```
Comment (animeshsingh) [inline samples/kfp-user-guide/README.md:152]:
```suggestion
## Run Pipelines
```

**Comment by ckadner** [comment]
@animeshsingh -- regarding the verb tense in the section headings, I remember that during our technical writing lessons they taught us to use the gerund form, i.e.

https://writing.stackexchange.com/questions/1545/verb-tense-for-technical-document-titles
> ### Which verb tense to use for technical document titles
> - Continuous (-ing) form: (e.g. "Creating a Cluster", "Creating a Database", ...)
> - Descriptive form: (e.g. "Cluster creation", "Database creation", ...)
> - Imperative form (e.g. "Create a Cluster", "Create a Database", ...)
> 
> ### Accepted Answer:
> `-ing`. I am a tech writer, and this is typically how we word titles of sections that focus on how to accomplish a task.

Or this one:

http://colelearning.net/rw_wb/module4/page14.html

**Pull request merged**